parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -240,6 +240,23 @@ async def synchronization_step(
         # Fetch and lock
         async with self.local_storage.lock_manifest(entry_id) as local_manifest:
 
+            # Go through the parent chain
+            current_manifest = local_manifest
+            while not is_workspace_manifest(current_manifest):
+                parent_manifest = await self.local_storage.get_manifest(current_manifest.parent)
```

Comment:
So we access each parent manifest here, could it be possible that the manifest is no longer present in the local_storage ? (currently we keep all the manifest in the local storage, but is this a strong guarantee or just an implementation detail that could silently break the code here if changed ?...)

And could it be possible the parent manifest has never been in the local storage ? for instance when using `WorkspaceFS.sync_by_id()` ?